cycledash/static/scss/examine.scss

```diff
@@ -12,39 +12,22 @@ a, a:hover {
   color: #000;
   text-decoration: none;
 }
-body {
-  font-size: 16px;
-  line-height: 1;
-  margin: 8px;
-}
 
-/* VCFTable */
+/*------[ VCF Table ]------*/
 .vcf-table {
-  padding-top: 20px;
+  @extend .table, .table-striped, .table-condensed;
```

Comment:
It's a matter of preference/style. So, I like extending Bootstrap classes because it lends to cleaner markup and helps give you more context/control in your stylesheet. It also lets you keep the classes in your markup semantic. For example, you can just name an element what it is (ie `.vcf-table`) and leave the rest to the stylesheet. Also, changing the styling of this object means updating the css, and not fiddling with the markup itself.

For the aside, when you extend class `.table` into `.vcf-table`, `.vcf-table` is now added into the css wherever the `.table` class is being defined. The styles aren't repeated (if that's what you meant?):
![screen shot 2015-05-22 at 12 29 45 pm](https://cloud.githubusercontent.com/assets/5685044/7775241/691a20dc-007f-11e5-9186-23b9261f7ad3.png)